src/main/java/com/synopsys/integration/alert/channel/azure/boards/web/AzureOauthCallbackController.java

```diff
@@ -0,0 +1,103 @@
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.google.gson.Gson;
+import com.synopsys.integration.alert.channel.azure.boards.AzureBoardsChannelKey;
+import com.synopsys.integration.alert.channel.azure.boards.descriptor.AzureBoardsDescriptor;
+import com.synopsys.integration.alert.channel.azure.boards.service.AzureBoardsProperties;
+import com.synopsys.integration.alert.channel.azure.boards.storage.AzureBoardsCredentialDataStoreFactory;
+import com.synopsys.integration.alert.common.enumeration.ConfigContextEnum;
+import com.synopsys.integration.alert.common.exception.AlertException;
+import com.synopsys.integration.alert.common.persistence.accessor.ConfigurationAccessor;
+import com.synopsys.integration.alert.common.persistence.accessor.FieldAccessor;
+import com.synopsys.integration.alert.common.persistence.model.ConfigurationModel;
+import com.synopsys.integration.alert.common.persistence.util.ConfigurationFieldModelConverter;
+import com.synopsys.integration.alert.common.rest.ProxyManager;
+import com.synopsys.integration.alert.common.rest.ResponseFactory;
+import com.synopsys.integration.alert.web.controller.BaseController;
+import com.synopsys.integration.azure.boards.common.http.AzureHttpService;
+
+@RestController
+@RequestMapping(AzureOauthCallbackController.AZURE_OAUTH_CALLBACK_PATH)
+public class AzureOauthCallbackController {
+    public static final String AZURE_OAUTH_CALLBACK_PATH = BaseController.OAUTH_CALLBACK_PATH + "/azure";
+    private Logger logger = LoggerFactory.getLogger(AzureOauthCallbackController.class);
+    private ResponseFactory responseFactory;
+    private final Gson gson;
+    private final AzureBoardsChannelKey azureBoardsChannelKey;
+    private final AzureBoardsCredentialDataStoreFactory azureBoardsCredentialDataStoreFactory;
+    private final ProxyManager proxyManager;
+    private final ConfigurationAccessor configurationAccessor;
+    private final ConfigurationFieldModelConverter configFieldModelConverter;
+
+    @Autowired
+    public AzureOauthCallbackController(ResponseFactory responseFactory, Gson gson, AzureBoardsChannelKey azureBoardsChannelKey,
+        AzureBoardsCredentialDataStoreFactory azureBoardsCredentialDataStoreFactory, ProxyManager proxyManager, ConfigurationAccessor configurationAccessor,
+        ConfigurationFieldModelConverter configFieldModelConverter) {
+        this.responseFactory = responseFactory;
+        this.gson = gson;
+        this.azureBoardsChannelKey = azureBoardsChannelKey;
+        this.azureBoardsCredentialDataStoreFactory = azureBoardsCredentialDataStoreFactory;
+        this.proxyManager = proxyManager;
+        this.configurationAccessor = configurationAccessor;
+        this.configFieldModelConverter = configFieldModelConverter;
+    }
+
+    @GetMapping
+    public ResponseEntity<String> oauthCallback() {
+        try {
+            logger.info("callback method called");
```

Comment:
This looks like it was added for testing, we may want to log something similar at DEBUG.